Summers_James/detail.html

```diff
@@ -0,0 +1,32 @@
+<%inherit file="./catalog/templates/app_base.htm" />
+
+
+
+<%block name="content">
+<div class="persCata">
+    <h1 style="left:45%; position: relative; top:18px;">${product.name}</h1>
+    <div style = "position:absolute; top: 70px; left: 20px;">
+        <ul style="display:flex; flex-direction: column; list-style: none;">
+        %for counter, im in enumerate(images):
```

Comment:
I had to look up enumerate but that is a way cool built in function, really concise _and_ gives you an automatic counter. I'd use this wherever else you loop and need a counter.